toolkit/tools/imagegen/installutils/installutils.go

```diff
@@ -1485,16 +1485,32 @@ func ProvisionUserSSHCerts(installChroot safechroot.ChrootInterface, username st
 	const squashErrors = false
 	const authorizedKeysTempFilePerms = 0644
 	const authorizedKeysTempFile = "/tmp/authorized_keys"
-	const sshDirectoryPermission = "0700"
+	const sshDirectoryPermission = 0700
```

Comment:
nit: `0o700`.

This just makes it more explicit that you are specifying an octal number.